lib/common.bash

```diff
@@ -73,7 +79,12 @@ extract_kata_env(){
 # Checks that processes are not running
 check_processes() {
 	extract_kata_env
-	general_processes=( ${PROXY_PATH} ${HYPERVISOR_PATH} ${SHIM_PATH} )
+	vsock_configured=$($RUNTIME_PATH kata-env | awk '/UseVSock/ {print $3}')
```

Comment:
you should also check if `vsock` is supported on the environment as sometimes the setting is enabled in the config file,  but the `vsock` module is not loaded. 
`vsock_supported=$($RUNTIME_PATH kata-env | awk '/SupportVSock/ {print $3}')`